modules/claims_api/spec/requests/v2/veterans/intent_to_files_request_spec.rb

```diff
@@ -145,7 +167,7 @@
 
               parsed_response = JSON.parse(response.body)
               expect(response.status).to eq(200)
-              expect(parsed_response['id']).to eq('2')
+              expect(parsed_response['data']['id']).to eq('2')
```

Comment:
you'll probably have to revert this as well when removing the `, root: :data` part... again, on me - apologies.